dom/timer/timer-dom.js

```diff
@@ -1,2 +1,57 @@
 console.log("time to start the timer!");
 
+const timer = document.querySelector('p');
```

Comment:
This works for now because you only have one <p> in your document but it's not good practice because it's not scalable. Since you want a timer to appear in only one place, using an ID is better practice.